packages/io/svg-deserializer/svgElementHelpers.js

```diff
@@ -108,7 +108,7 @@ const svgSvg = function (element, { customPxPmm }) {
   if ('HEIGHT' in element) { obj.height = element.HEIGHT }
   if ('VIEWBOX' in element) {
     const list = element.VIEWBOX.trim()
-    const exp = new RegExp('([\\d\\.\\-]+)[\\s,]+([\\d\\.\\-]+)[\\s,]+([\\d\\.\\-]+)[\\s,]+([\\d\\.\\-]+)', 'i')
+    const exp = /([\d.-]+)[\s,]+([\d.-]+)[\s,]+([\d.-]+)[\s,]+([\d.-]+)/i
```

Comment:
Why make these changes to the regular expressions? saving space?

Actually, these expressions should be created once at the file level, and reused. That would improve performance.

Reply:
standardx complained that new RegExp should not be used

Reply:
reusing regex with global flag can be dangerous if not careful
https://medium.com/codesnips/js-careful-when-reusing-regex-636b92c6bf07
https://stackoverflow.com/questions/1520800/why-does-a-regexp-with-global-flag-give-wrong-results

but it looks those 2 are not global, there is no /g flag so we should be ok